File: docs/examples/py/src/access_control_remove_token.py
from reduct import Client, Permissions


async def remove_token():
    # Create a client with the base URL and API token
    with Client("http://localhost:8383", api_token="my-token") as client:
        # Remove the token with the name "token-to-remove"
        await client.remove_token("token-to-remove")


if __name__ == "__main__":
    import asyncio

    asyncio.run(remove_token())

File: versioned_docs/version-1.20.x/examples/go/src/access_control_remove_token.go
package main

import (
	"context"
	reduct "github.com/reductstore/reduct-go"
)

func main() {
	// Create a client and use the base URL and API token
	client := reduct.NewClient("http://localhost:8383", reduct.ClientOptions{
		APIToken: "my-token",
	})

	// Remove the token with the name "token-to-remove"
	err := client.RemoveToken(context.Background(), "token-to-remove")
	if err != nil {
		panic(err)
	}

}
